### PR TITLE
enhancement/issue-12-standarizeAPI

### PR DESCRIPTION
**Pull request by cxoltero**
kenzanlabs/keystone#7
issue #12

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
Can one of the admins verify this patch?

**Comment by obuckley-kenzan** [comment]
@cxoltero per the CONTRIBUTING.md, please reference using `#` the issue this resolves

**Comment by obuckley-kenzan** [comment]
ok to test

File: src/index.js [hunk 1]
```diff
@@ -19,38 +19,39 @@ var config = {
...
-      .pipe(function() {
-        return gulpIf(config.addSourceMaps, sourcemaps.write('maps'));
-      });
+module.exports = {
+  compileLESS: function(options) {
+    if (options) {
+      handyman.log('Merging custom configuration');
+      config = handyman.mergeConf(config, options);
+    }
+    return compileLESS();
```
Comment (obuckley-kenzan) [inline src/index.js]: Could we just do `return makePipe();` here? Just curious the value in the additional function call / wrapper of `compileLESS`?

cc: @paul-barry-kenzan

Comment (paul-barry-kenzan) [inline src/index.js]: @cxoltero @obuckley-kenzan 
I have to agree with Owen here. The `makePipe()` should be sufficient.

**Comment by obuckley-kenzan** [comment]
@cxoltero 
Please update the README

**Comment by cxoltero** [comment]
@obuckley-kenzan 
README is updated on this PR, commit 892ce06, PR should be ready.

**Comment by obuckley-kenzan** [comment]
oops, I missed it, thanks!

**Pull request merged**